manifests/crc/default-db-included-installation/openshift/kruize-crc-openshift.yaml

```diff
@@ -209,6 +209,18 @@ spec:
         name: kruize
     spec:
       serviceAccountName: kruize-sa
+      initContainers:
```

Comment:
As discussed today, continuing with `initContainer` approach only. 
I have tried checking the DB  connection by login in the pod. Since, we don't have `ping` , `curl` or `netcat` available within the pod, I logged into the DB with the name `kruizeDB` and was able to connect successfully.
```
saakhan:autotune$ kubectl -n monitoring exec -it kruize-564fcbbc66-pbw8g -c wait-for-postgres -- /bin/bash
root@kruize-564fcbbc66-pbw8g:/# psql -h postgres-service -U admin -d kruizeDB
Password for user admin: 
psql (15.2 (Debian 15.2-1.pgdg110+1))
Type "help" for help.

kruizeDB=#
```